packages/react-native/ReactAndroid/api/ReactAndroid.api

```diff
@@ -6720,9 +6720,9 @@ public class com/facebook/react/views/imagehelper/ImageSource {
 	public final fun getSize ()D
 	public final fun getSource ()Ljava/lang/String;
 	public static final fun getTransparentBitmapImageSource (Landroid/content/Context;)Lcom/facebook/react/views/imagehelper/ImageSource;
-	public final fun getUri ()Landroid/net/Uri;
+	public fun getUri ()Landroid/net/Uri;
 	public fun hashCode ()I
-	public final fun isResource ()Z
+	public fun isResource ()Z
```

Comment:
Just a small heads up that this is not the right fix. This file is generated from the Kotlin code, so the code needs to be updated as well. I'll take care of it

Reply:
@cortinico Can you also explain how this file gets generated for future reference ?

Reply:
This file gets generated with `./gradlew apiDump` (see https://github.com/Kotlin/binary-compatibility-validator)
Also we regenerate the file internally with Buck so even if you edit it, we'll always regenerate it once we import the PR

Reply:
So ideally it should run after your [PR](https://github.com/facebook/react-native/pull/46093) .
But didn't happen .

Reply:
How about a "thank you" also there @deepanshushuklad11 

Thanks @cortinico

Reply:
@bradfloodx Yes a big thanks to @cortinico.
Just wanted to clarify my doubts.